Update docker compose file and instructions

kenwalker opened the pull request:
These changes allowed me to be able to run the Ork within a Docker container.  I switched to MySql from MariaDb since that seems to be what is on production.  This includes logging in, updating or creating new entities.  I also tried the Ork Mobile against it and you'll see the small change in the index.php file for the service to allow it to work on dev http.

The only odd thing was that I had to set the following mode in SQL otherwise there are several places where time() values are sent and rejected, and all creates attempt to set the AUTO_INCREMENT field to "".  Also rejected.

`SET GLOBAL sql_mode = '';`

kenwalker commented on README.md:
````diff
@@ -27,11 +27,30 @@ Copy `config.dev.php` to `config.php`. Make sure to change the admin email to yo
 
-You will still need to hydrate the database from a backup. If you have other environments using port 80 you may need to change the exposed port in the docker-compose file to keep from conflicting. Same goes for 3306 for the database. Out of the box and without port conflicts you should be able to run the command above, import the DB to the 'ork' database, and begin work. You may want to adjust th ...
+```
+mysql -P 3306 --protocol=tcp -h localhost -u ork  -p ork < ~/Downloads/2019.02.28.02.00.amtgard_ork3production.sql
````
Darn, thought I left that out of the command line.  I guess because it was prompting me for password I thought it was all good.

kenwalker commented on README.md:
````diff
@@ -27,11 +27,30 @@ Copy `config.dev.php` to `config.php`. Make sure to change the admin email to yo
+```
+This will take a while but will hydrate the database with the backup that was downloaded and extracted above.
+```
+mysql -P 3306 --protocol=tcp -h localhost -u root  -p ork 
````
Same as above, was mistake. Good catch.

kenwalker commented:
@mathus13 Fixed in follow on commit.  Can I assume you tried this whole set of steps just to confirm some other human has had success?  Cheers.

mathus13 commented:
I first commented on my work computer, so I didn't want to load it there.
The 'ork' in the original was referencing the database name, not the password like I thought, so lets add 'ork' back in there 
`mysql -P 3306 --protocol=tcp -h localhost -u ork  -psecret ork < ork.sql`

I found line 43 required root access to run DROP
`mysql -P 3306 --protocol=tcp -h localhost -u root  -proot ork < ~/Downloads/2019.03.12.02.00.amtgard_ork3production.sql`

With those 2 changes (and some environmental stuff) I was golden

kenwalker commented:
Thanks @mathus13.  I've just updated all the sql lines to use the root/root combination and cleaned up the doc around the lines.  I ran through a clean install and it seems good.  Thanks for verifying it worked for you.
